commands/dataplatform/dataplatform_e2e_integration_test.go

```diff
@@ -1,3 +1,4 @@
+// build +integration
```

Comment:
if we're separating integration test files from unit test files, then let's also use a naming convention for the files: `dataplatform_integration_test.go` instead of `dataplatform_e2e_integration_test.go`?

Reply:
also `// +build integration`